### DISCUSSION TITLE
Storefront not found when editing header or footer

### DISCUSSION THREAD

**Discussion by naberna** [swellstores/community#431] [Help] [answered]
Hello,

I am writing behalf of my store 'Just Check': (just-check.swell.store)

When I edit header or footer I am getting this error: 'Storefront not found'

Also, I wonder how can I add validation to the user password.

**Comment by naberna** [comment]
@logeshswell can you please help?

**Reply by logeshswell** [reply]
@naberna Can you share a screenshot of the error?

**Comment by logeshswell** [comment]
> how can I add validation to the user password?

What kind of validation do you want to add?

**Reply by naberna** [reply]
Dear @logeshswell,

See the screenshot: Store not found

<img width="1512" alt="Screenshot 2023-11-15 at 07 09 40" src="https://github.com/swellstores/community/assets/73580153/da1ccf18-3a19-465a-ba63-509ce8122c34">

I dont know if it is possible but I would like to embed a video and add a visa and mastercard picture in checkout page. 

Kind regards,

**Reply by logeshswell** [reply, marked as answer]
I filed a bug ticket for this issue and will keep you posted if I get some updates.

**Reply by logeshswell** [reply]
In the meantime, you can update the menu details under **Navigation** tab on the Storefront details page.
